yarn.lock

```diff
@@ -27,6 +27,22 @@
   dependencies:
     axe-core "^4.7.0"
 
+"@babel/cli@^7.21.0":
```

Comment:
I don't think we need so many babel versions. `yarn deduplicate && yarn install` can help out here.